ReactWindows/ReactNative.Shared/Views/Image/ReactImageLoadEvent.cs

```diff
@@ -72,6 +90,8 @@ public override string EventName
                         return "topLoad";
                     case OnLoadEnd:
```

Comment:
What do we do with this one, then? Doesn't seem to be used.
Semantically "onLoad" marks the end, anyway.

[edit]: Based on the comments of the JS counterpart, onLoadEnd should be called in both cases (error & success).
So order for success:
- onLoadstart, onLoad, onLoadEnd
for failure:
- onLoadStart, onError, onLoadEnd 

(not sure about the order of onLoadEnd vs onXXX, perhaps peeking into Android code would inspire somehow.)

Reply:
updated to fire both events as on android